syz-manager/html.go

```diff
@@ -334,42 +334,24 @@ func (mgr *Manager) httpCoverCover(w http.ResponseWriter, r *http.Request, funcF
 		coverFilter = mgr.coverFilter
 	}
 
-	if funcFlag == DoRawCoverFiles {
-		if err := rg.DoRawCoverFiles(w, progs, coverFilter); err != nil {
-			http.Error(w, fmt.Sprintf("failed to generate coverage profile: %v", err), http.StatusInternalServerError)
-			return
-		}
-		runtime.GC()
```

Comment:
Maybe keep this call, but run it after the callback?
I don't have a strong opinion though.